logrotation-k8env/logrotate/config/startup.py

```diff
@@ -0,0 +1,65 @@
+# /usr/bin/python3
```

Comment:
This file is not related to logrotate config. In fact this uses logrotate config. Hence needs to be moved out of logrotate.

Reply:
Merged the startup.py scripts from ./logrotate/config and ./logrotate/cron dir & move it to /startup/logrotate_setup.py.